stack.yaml

```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```

Comment:
I see so many of these nowadays, it's almost mechanical to add them to a project :P

I'll break it out into a separate PR